integration_tests/src/main/python/delta_lake_test.py

```diff
@@ -100,7 +100,8 @@ def setup_tables(spark):
 @ignore_order(local=True)
 @pytest.mark.parametrize("reader_confs", reader_opt_confs_no_native, ids=idfn)
 @pytest.mark.parametrize("mapping", column_mappings, ids=idfn)
-def test_delta_read_column_mapping(spark_tmp_path, reader_confs, mapping):
+@pytest.mark.parametrize("deletion_vector_conf", deletion_vector_conf, ids=idfn)
+def test_delta_read_column_mapping(spark_tmp_path, reader_confs, mapping, deletion_vector_conf):
```

Comment:
Why do we make a read test as xfail for databricks 14.3? Wouldn't we want a read version of `deletion_vector_conf` that  does not have an xfail in it?